def failed(n):
    
    return "Fail to solve but pull-request.....ㅠ-ㅠ"